src/main/java/com/upserve/uppend/cli/CommandFileStoreBenchmark.java

```diff
@@ -0,0 +1,80 @@
+package com.upserve.uppend.cli;
+
+import com.upserve.uppend.blobs.*;
+import com.upserve.uppend.cli.benchmark.*;
+import org.slf4j.Logger;
+import picocli.CommandLine;
+
+import java.lang.invoke.MethodHandles;
+import java.nio.file.*;
+import java.util.Random;
+import java.util.concurrent.Callable;
+import java.util.stream.IntStream;
+
+@SuppressWarnings({"WeakerAccess", "unused"})
+@CommandLine.Command(
+        name = "filestore",
+        description = "Run file store benchmark",
+        showDefaultValues = true,
+        synopsisHeading = "%nUsage: uppend ",
+        parameterListHeading = "%nParameters:%n",
+        optionListHeading = "%nOptions:%n",
+        commandListHeading = "%nCommands:%n",
+        descriptionHeading = "%n",
+        footerHeading = "%n"
+)
+public class CommandFileStoreBenchmark implements Callable<Void> {
+    private static final Logger log = org.slf4j.LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+
+    @CommandLine.Parameters(index = "0", description = "Store path")
+    Path path;
+
+    @CommandLine.Option(names= {"-n", "--virtual-files"}, description = "the number of virtual files to use")
+    int nfiles = 32;
+
+    @CommandLine.Option(names = {"-b", "--buffer-size"}, description = "Buffer Size (small|medium|large)")
+    BufferSize bufferSize = BufferSize.medium;
+
+    @CommandLine.Option(names = {"-s", "--size"}, description = "Benchmark size (nano|micro|small|medium|large|huge|gigantic)")
```

Comment:
Reminds me of this:
https://en.wikipedia.org/wiki/Intel_Memory_Model#Memory_models
The combinations were for code and/or data to be less than/greater than 64KB.